handlers/create.go

```diff
@@ -59,12 +65,13 @@ func CreateMachine(event *events.Event, apiClient *client.RancherClient) error {
 		return err
 	}
 
-	errString := ""
-	go logProgress(readerStdout, readerStderr, publishChan, machine, event, &errString)
+	errChan := make(chan string, 1)
+	go logProgress(readerStdout, readerStderr, publishChan, machine, event, errChan)
 
 	err = command.Wait()
 
 	if err != nil {
+		errString := <-errChan
```

Comment:
That's what i wanted to talk about.  It should not if you close the channel, right?